content/millennium/dstu2/proprietary-codes.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Proprietary Codes | DSTU2 API
+---
+
+# Proprietary Codes
+
+* TOC
+{:toc}
+
+## Overview
+
+Cerner's implementation of the HL7<sup>®</sup> DSTU2 FHIR<sup>®</sup> standard allows Millennium proprietary code values
+to be used in addition to standard value set codes on a limited basis. This allows developers to read and write data
```

Comment:
"on a limited basis." is distinct from the R4 page. not sure if we'd want to remove it there or add it here

Reply:
talked offline, keeping this